@kiva/kv-components/vue/KvLocationSelector.vue

```diff
@@ -0,0 +1,236 @@
+			<legend class="tw-w-full md:tw-mb-1">
+				<button
+					class="tw-transition-all tw-flex tw-items-center md:tw-justify-between tw-w-full"
+					@click="toggleRegion(option.region)"
+				>
+					<kv-material-icon
+						:icon="isOpenRegion(option.region) ? mdiChevronDown : mdiChevronRight"
+						class="tw-h-1.5 tw-w-1.5 md:tw-h-3 md:tw-w-3 tw-mr-1 tw-transition-all md:tw-order-2"
+					/>
+					<span class="md:tw-order-1 md:tw-text-subhead md:tw-flex tw-items-center">
+						<span class="tw-w-4 tw-text-action tw-text-base tw-text-right tw-mr-1 tw-hidden md:tw-inline">
+							{{ numberByRegion(option.region) !== 0 ? `(${numberByRegion(option.region)})` :'' }}
+						</span>
+						{{ getItemLabel(option) }} <span class="md:tw-hidden">{{ getNumberLoans(option) }}</span>
+					</span>
+				</button>
+			</legend>
+			<menu
+				v-if="isOpenRegion(option.region)"
+				class="tw-flex tw-flex-col tw-gap-1.5 tw-my-1.5"
+			>
+				<li class="md:tw-hidden">
+					<button
+						class="tw-text-link"
+						@click="toggleSelectAll(getItems(option.countries))"
+					>
+						{{ isAllSelected(getItems(option.countries)) ? 'Deselect' : 'Select' }} all
+					</button>
+				</li>
+				<div class="tw-grid tw-grid-cols-1 md:tw-grid-cols-3 tw-gap-1.5 md:tw-gap-2.5 md:tw-pl-5">
+					<li class="tw-hidden md:tw-block">
+						<kv-checkbox
+							v-model="allSelectedModel"
+							:value="`selected-all-${option.region}`"
+							@change="toggleSelectAll(getItems(option.countries))"
+						>
+							{{ isAllSelected(getItems(option.countries)) ? 'Deselect' : 'Select' }} all
+						</kv-checkbox>
+					</li>
+					<li
+						v-for="(item, i) in getItems(option.countries)"
+						:key="i"
+					>
+						<kv-checkbox
+							v-model="selected"
+							:value="item.value"
+							:disabled="item.disabled"
+							@change="updateLocation"
+						>
+							{{ item.title }}
+						</kv-checkbox>
+					</li>
+				</div>
+			</menu>
+		</fieldset>
+		<div class="tw-hidden md:tw-flex tw-gap-2 tw-justify-end tw-mt-1.5">
+			<button
+				class="tw-text-link"
+				@click="resetCountries"
+			>
+				Reset country selection
+			</button>
+
+			<kv-button
+				@click="closeModal"
+			>
+				See {{ totalLoans }} loans
+			</kv-button>
+		</div>
+	</form>
+</template>
+
+<script>
+import {
+	ref,
+	toRefs,
+} from 'vue-demi';
+import { mdiChevronDown, mdiChevronRight } from '@mdi/js';
+import KvMaterialIcon from './KvMaterialIcon.vue';
+import KvCheckbox from './KvCheckbox.vue';
+import KvButton from './KvButton.vue';
+
+export default {
+	components: {
+		KvMaterialIcon,
+		KvCheckbox,
+		KvButton,
+	},
+	props: {
+		/**
+		 * The regions with countries used to build the checkbox lists. Expected format:
+		 * [{
+		 *   region: 'region',
+		 *   numLoansFundraising: 1,
+		 *   countries: [{
+		 *     name: 'name',
+		 *     isoCode: 'US',
+		 *     numLoansFundraising: 1,
+		 *   }]
+		 * }]
+		 */
+		regions: {
+			type: Array,
+			default: undefined,
+		},
+		activeIsoCodes: {
```

Comment:
This can be a follow-up task, please add a ticket, but we'll need this component to watch `activeIsoCodes` for changing the selected values based on this prop changing. The use case:

1. User selects a country not in US
2. User selects US category
3. Filters get updated to have country selection that matches category
4. Location filter should show the updated filter (without user having changed the location filter)

There are other cases, but that's the general idea.